<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="paso1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Fusi贸n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paso2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Rectificaci贸n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paso3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Segmentaci贸n</string>
   </property>
  </widget>
  <widget class="QToolButton" name="editarSegmentacion">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar segmentaci贸n</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
